src/api/operator/operatorApi.ts

```diff
@@ -162,3 +168,57 @@ export async function getFeeQuote(networkId: Network, tokenAddress: string): Pro
     return response.json()
   }
 }
+
+/**
+ * Gets a single order by id
+ */
+export async function getOrder(params: GetOrderParams): Promise<RawOrder> {
+  const { networkId, orderId } = params
+
+  console.log(`[getOrder] Fetching order id '${orderId}' on network ${networkId}`)
+
+  const queryString = `/orders/${orderId}`
+
+  const response = await _get(networkId, queryString)
+
+  if (!response.ok) {
+    throw new Error(`Request failed: [${response.status}] ${await response.text()}`)
+  }
+
+  const order: RawOrder = await response.json()
+
+  return order
+}
+
+/**
+ * Gets a list of orders
+ * Optional filters:
+ *  - owner: address
+ *  - sellToken: address
+ *  - buyToken: address
+ */
+export async function getOrders(params: GetOrdersParams): Promise<RawOrder[]> {
+  const { networkId, owner, sellToken, buyToken } = params
+
+  console.log(
+    `[getOrders] Fetching orders on network ${networkId} with filters: owner=${owner} sellToken=${sellToken} buyToken=${buyToken}`,
+  )
+
+  const queryString =
+    '/orders/' +
+    buildQueryString([
+      ['owner', owner],
+      ['sellToken', sellToken],
+      ['buyToken', buyToken],
+    ])
+
+  const response = await _get(networkId, queryString)
+
+  if (!response.ok) {
+    throw new Error(`Request failed: [${response.status}] ${response.body}`)
+  }
+
+  const orders: RawOrder[] = await response.json()
+
+  return orders
```

Comment:
dont want to just `return response.json()` ?

Reply:
uber nit that was hehe

Reply:
I definitely can, and I think I tried but typescript.
(Or did I? Don't know, will try again)

Reply:
It worked \o/